.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
     - main
+    - planning-1.0-release
```

Comment:
@SemyonSinchenko - are you okay to have this here momentarily until this branch gets merged to the main? Let me know your preference.

Reply:
At the moment I'm triggering it manually from time to time and it is OK for me. But let's have it here and delete it before merging into main.

Reply:
Noted. Thanks. As you can see the latest changes I pushed after marking those deps as optional has succeeded in the workflow. There seems to be a conflict with the Makefile due to another PR that might have been merged. I am trying to resolve it right now.